haas/client/auth.py

```diff
@@ -0,0 +1,17 @@
+import requests
+
+
+def db_auth(username, password):
+    """ For database as authentication backend, this function prepares
+    the default http client session using requests module.
```

Comment:
For the same reason (no. 2) that you mentioned I wanted to call it db_auth, so distinguish it from other authentication back-ends. This one clarifies that this back-end resides in the database.